### DISCUSSION TITLE
Add STRIDE classification of vulnerabilities

### DISCUSSION THREAD

**Discussion by MichaelGrafnetter** [Syslifters/sysreptor#305] [General]
It would be nice to be able to optionally add [STRIDE](https://en.wikipedia.org/wiki/STRIDE_model) classification of vulnerabilities.

Sample editor for a vulnerability:

- [X] **S**poofing
- [ ] **T**ampering
- [ ] **R**epudiation
- [ ] **I**nformation disclosure
- [ ] **D**enial of service
- [X] **E**levation of privilege

Corresponding presentation in a vulnerability report: **<ins>S</ins>** T R I D **<ins>E</ins>**

**Comment by aronmolnar** [comment]
This is something you could easily build yourself by creating a finding field of type ["Object"](https://docs.sysreptor.com/designer/field-types/#object), which then holds six fields of type ["Boolean"](https://docs.sysreptor.com/designer/field-types/#boolean).

**Reply by MichaelGrafnetter** [reply]
True. It's just an idea and time-savers working out-of-the-box is what I like about this project the most.
